app/screens/send-bitcoin-screen/use-send-payment.ts

```diff
@@ -26,18 +27,24 @@ export const useSendPayment = (
 ): UseSendPaymentResult => {
   const [intraLedgerPaymentSend, { loading: intraLedgerPaymentSendLoading }] =
     useIntraLedgerPaymentSendMutation()
```

Comment:
Although the current way may work, I think it is cleaner to pass the refetch queries to the hook like 
```
useIntraLedgerPaymentSendMutation({
    refetchQueries: [MainAuthedDocument]
})
```

Reply:
I've tried that, but you can't pass this variable here

Reply:
maybe give it a try, I may have had the type wrong

Reply:
What wasnt working for you? I dont have any type errors when I do that

Reply:
was not working but working now. probably ts cache issue :/